src/main/java/uk/gov/companieshouse/confirmationstatementapi/controller/ConfirmationStatementController.java

```diff
@@ -58,4 +60,15 @@ public ConfirmationStatementController(ConfirmationStatementService confirmation
         return serviceResponse.<ResponseEntity<Object>>map(ResponseEntity::ok)
                 .orElseGet(() -> ResponseEntity.notFound().build());
     }
+
+    @GetMapping("/{confirmation_statement_id}/validation-status")
+    public ResponseEntity<Object> getValidationStatus(@PathVariable("confirmation_statement_id") String submissionId) {
+        try {
+            ValidationStatusResponse validationStatusResponse = confirmationStatementService.areTasksComplete(submissionId);
+            return ResponseEntity.ok().body(validationStatusResponse);
+        } catch (CompanyNotFoundException e) {
```

Comment:
i'd change the exception type to Exception just to make sure we catch any other exceptions apart from CompanyNotFoundException. You could add it after this block but looks like they'll both be printing the same message so probably better just to change the existing catch block.

Reply:
actually, stick the extra catch(Exception e) after this block and have a generic message. The message in CompanyNotFoundException is relating to submission not found, perhaps change this message to say Could not find company.